scripts/styleguide.config.js

```diff
@@ -98,7 +97,10 @@ const allSections = [
             '../src/components/theme/index.js',
             '../src/components/thumbnail-card/ThumbnailCard.js',
             '../src/components/toggle/Toggle.js',
-            '../src/components/tooltip/Tooltip.js',
+
+            '../src/components/button/Button.tsx',
```

Comment:
Had to add these back because other non-migrated components in the styleguide depend on it